TaskManagerTLA.DAL/Repositories/IdentityRep/RolesRepository.cs

```diff
@@ -82,28 +61,19 @@ public async Task<ApplicationRole> GetItemByIdAsync(string itemId)
                 return dataBase.Roles.Find(itemId);
             });
         }
-
-        public async Task<ApplicationRole> FindItemAsync(Expression<Func<ApplicationRole, bool>> predicate)
+        public async Task<ApplicationRole> GetItemByNameAsync(string itemName)
         {
             return await Task.Run(() =>
             {
-                return dataBase.Roles.FirstOrDefault(predicate);
+                return dataBase.Roles.Where(p => p.Name == itemName).Include(p => p.Users).FirstOrDefault();
```

Comment:
Для чого Task.Run якщо є Async() методи?
```suggestion
                return dataBase.Roles.Where(p => p.Name == itemName).Include(p => p.Users).FirstOrDefaultAsync();
```